pgbot/__init__.py

```diff
@@ -450,29 +459,31 @@ async def handle_message(msg: discord.Message):
     if msg.type == discord.MessageType.premium_guild_subscription:
         await emotion.server_boost(msg)
 
-    if msg.content.startswith(common.PREFIX):
-        ret = await commands.handle(msg)
+    mentions = f"<@!{common.bot.user.id}>", f"<@{common.bot.user.id}>"
+
+    if (
+        msg.content.startswith(common.COMMAND_PREFIX)
+        or msg.content.startswith(mentions)
+        and msg.content not in mentions  # ignore normal pings
```

Comment:
I have to look up the precedence of `X or Y and Z` everytime... so maybe add parenthesis here for clarity sake so that the logic is immediately clear

Reply:
`and` *always* precedes over `or`, and `not` over and. All I'm doing here is short-circuiting to block bare mentions of the bot from triggering the command system.